pyaedt/modules/PostProcessor.py

```diff
@@ -5018,17 +5093,38 @@ def get_field_summary_data(self, sweep_name=None, design_variation={}, intrinsic
         return out_dict
 
     @pyaedt_function_handler()
-    def export_csv(self, filename, sweep_name=None, design_variation={}, intrinsic_value=""):
-        if not sweep_name:
-            sweep_name = self._app.nominal_sweep
+    def export_csv(self, filename, setup_name=None, design_variation={}, intrinsic_value=""):
+        """
+        Get the field summary output computation.
+
+        Parameters
+        ----------
+        filename : str
+            Path and filename to write the output file to.
+        setup_name : str, optional
+            Setup name to use for the computation. The
+            default is ``None``, in which case the nominal variation is used.
+        design_variation : dict, optional
+            Dictionary containing the design variation to use for the computation.
+            The default is  ``{}``, in which case the nominal variation is used.
+        intrinsic_value : str, optional
+            Intrinsic values to use for the computation. The default is ``""``.
```

Comment:
Same here about indicating the behavior that occurs when the default is used.

Reply:
```suggestion
            Intrinsic values to use for the computation. The default is ``""``, 
            suitable when no frequency needs to be selected.
```